site/public/css/bootstrap.css

```diff
@@ -201,6 +201,12 @@ fieldset[disabled] .btn-primary.focus {
     background-color: var(--btn-default-white);
 }
 
+.btn-course-material-folder-set-time {
+    line-height: 15px;
+    height: 30px;
+    margin: 1px;
+}
```

Comment:
I believe that just adding a little bit of margin on the bottom of each row would be more appropriate than adjusting the button itself.